source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -906,6 +913,13 @@ let from_json_list source_json_list =
         let location = JsonAst.Json.Util.to_location_exn node in
         Result.return (value, location))
   in
+  let json_string_member_with_location ~path key value =
+    json_exception_to_error ~path ~section:key (fun () ->
+        let node = JsonAst.Json.Util.member_exn key value in
+        let value = JsonAst.Json.Util.to_string_exn node in
+        let location = JsonAst.Json.Util.to_location_exn node in
+        Result.return (value, location))
+  in
```

Comment:
looks like https://github.com/facebook/pyre-check/pull/741 also adds this, so we will need to be careful when merging this.